src/job/job.py

```diff
@@ -737,11 +738,23 @@ def run_job_docker_compose(self, c):
             compose_profiles = self.job['definition'].get('compose_profiles', [])
             self.environment['PATH'] = os.environ['PATH']
 
-            cmds = ['docker-compose', '-f', compose_file_new, 'build']
+            if  self.job['definition'].get('enable_docker_build_kit', False) is True:
+                os.environ['DOCKER_BUILDKIT'] = '1'
+                os.environ['COMPOSE_DOCKER_CLI_BUILD']= '1'
+                c.collect('BUILDKIT is enabled during build!', show=True)
+                self.environment['DOCKER_BUILDKIT'] = '1'
```

Comment:
clean up unnecessary env

Reply:
I will clean up the os.environment ones.